README.md

```diff
@@ -87,6 +87,11 @@ for example
 $ docker run -it -v `pwd`:/src echidna echidna-test /src/examples/solidity/basic/flags.sol
 ```
 
+Nix users can install the lastest Echidna with:
```

Comment:
I think we should list Nix as the last option (maybe in its own subsection), to avoid confusing new users (they should use docker or the precompiled version)

Reply:
sure, I can move it below stack